python/test/t_Gibbs_std.py

```diff
@@ -1,14 +1,55 @@
 #! /usr/bin/env python
-
 import openturns as ot
 import openturns.testing as ott
-import math as m
+import math
 
 ot.TESTPREAMBLE()
 
 ot.RandomGenerator.SetSeed(0)
 
-# this analytical example is taken from "Bayesian Modeling Using WinBUGS" - Ioannis Ntzoufras
```

Comment:
Why removing this interesting comment?

Reply:
Maybe a mistake on my part? I will put it back in.

Reply:
Actually this comment was not removed, but moved in the script, so nothing to do here.